package.json

```diff
@@ -221,7 +221,8 @@
     "test": "jest",
     "package": "vsce package --yarn",
     "tpip:report": "ts-node scripts/tpip-reporter --header docs/tpip-header.md docs/third-party-licenses.json TPIP.md",
-    "lint:md": "markdownlint **/*.md -c ./.github/markdownlint.jsonc -i ./node_modules ./dist ./coverage ./tools"
+    "lint:md": "markdownlint **/*.md -c ./.github/markdownlint.jsonc -i ./node_modules ./dist ./coverage ./tools",
+    "check:links": "find . -name '*.md' -not -path './node_modules/*' -not -path './.github/*' -exec npx markdown-link-check -v -c ./.github/markdown-link-check.jsonc {} +"
```

Comment:
I get an error when I run this with the Windows cmd shell.:
```
....\vscode-cmsis-debugger>yarn check:links
yarn run v1.22.22
$ find . -name '*.md' -not -path './node_modules/*' -not -path './.github/*' -exec npx markdown-link-check -v -c ./.github/markdown-link-check.jsonc {} +
FIND: Parameter format not correct
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Ahh yes, `find `command behaves differently on Windows.  We might need better cross-platform alternatives.

Reply:
Do these probably help to implement the `find` part as CLI args to `markdown-link-check`?

Reply:
these = https://github.com/marketplace/actions/markdown-link-check#custom-variables

Reply:
Here is commit to address this feedback :
[check links script](https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/pull/264/commits/6c8408d0f46d3bad8e8a7115d32eedae33bebd27)

Reply:
Great, thanks! Works now in both git bash and cmd on Windows.